src/assets/css/main.css

```diff
@@ -276,7 +277,10 @@ hr {
 }
 
 .mini-overlay {
-  display: block;
```

Comment:
This feels like it doesn't belong in this PR?

Reply:
I can make it a separate PR if you think that's appropriate. Keeping it the old way makes the thumbnail icon show up incorrectly image thumbnails.

Reply:
Ah, I remember how I handled this now. I added these classes to the thing element wrapping it:

https://github.com/LemmyNet/lemmy-ui/pull/1570/files#diff-cb34074ed5916cfbee02ee325240476c1ee8dd515e9e4559ffff1eaeec039f53R50

I think this makes more sense, to absolutely position the wrapper element (link/button) and keep the icon itself just positioned naturally within it.